Features/HideProtectedPosts/functions.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Flynt\Features\HideProtectedPosts;
+
+add_action('pre_get_posts', function ($query) {
+    if (!$query->is_singular() && !is_admin()) {
+        $query->set('has_password', false);
```

Comment:
this works since 3.9? did you get it from here: https://stackoverflow.com/a/22971815

most articles about that and even the codex (https://codex.wordpress.org/Using_Password_Protection#Hiding_Password_Protected_Posts) are outdated then.